bootstrap/Makefile

```
@@ -62,6 +62,7 @@ docker/run: docker/build
 	-e AWS_PROFILE=$(AWS_PROFILE) \
 	-e AWS_SHARED_CREDENTIALS_FILE=/.aws/credentials \
 	-e INCLUDE=/ci-harness/$(PROJECT_NAME)/Makefile \
+	-e TARGET_PROJECT_NAME=$(PROJECT_NAME) \
```

Also, did you try this as just

```suggestion
	-e PROJECT_NAME=$(PROJECT_NAME) \
```

to avoid needing to define a new env var?

---

think I got it sorted without having to make more changes here to the docker env vars